modules/network-observability-dns-tracking.adoc

```diff
@@ -33,12 +33,15 @@ spec:
     ebpf:
       features:
        - DNSTracking           <1>
-      privileged: true         <2>
```

Comment:
flowcollector API has to be updated as suggested in RTT doc. Also PascalCase changes needs to be incorporated too

Reply:
@Amoghrd Ok I removed the privileged: true and added the recommendation for set sampling to 1